apps/web/src/pages/DirectivePage/Resources.tsx

```diff
@@ -1,12 +1,10 @@
 import { useIntl } from "react-intl";
-import ClipboardDocumentIcon from "@heroicons/react/24/outline/ClipboardDocumentIcon";
 import FolderOpenIcon from "@heroicons/react/24/outline/FolderOpenIcon";
-import ArrowDownOnSquareIcon from "@heroicons/react/24/outline/ArrowDownOnSquareIcon";
+import ArrowDownOnSquareIcon from "@heroicons/react/24/solid/ArrowDownOnSquareIcon";
 
 import { CardBasic, Heading, Link } from "@gc-digital-talent/ui";
-import { uiMessages, getLocale } from "@gc-digital-talent/i18n";
+import { getLocale } from "@gc-digital-talent/i18n";
 
-import decisionTree from "~/assets/img/Directive_landing_page_graphics_R1-02.webp";
 import managers from "~/assets/img/Managers_image.webp";
```

Comment:
Should we also delete this file? It seems like this was the only place it was being used.

Reply:
Good catch! Removed in [remove unused image](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11499/commits/238a2fd415aa6409466c9e4171bb9194201f7871)